lib/sprockets/base.rb

```diff
@@ -115,8 +115,8 @@ def find_all_linked_assets(*args)
     #
     #     environment['application.js']
     #
-    def [](*args)
-      find_asset(*args)
+    def [](*args, **options)
```

Comment:
The spockets-rails method call should not raise any warning since it is passing a hash as second argument, not keyword arguments.

Reply:
Ah, I see. It is because `find_asset` is doing the split.